build/yamls/base/controller.yml

```diff
@@ -138,6 +138,21 @@ webhooks:
     admissionReviewVersions: ["v1", "v1beta1"]
     sideEffects: None
     timeoutSeconds: 5
+  - name: "groupvalidator.antrea.io"
+    clientConfig:
+      service:
+        name: "antrea"
+        namespace: "kube-system"
+        path: "/validate/group"
+    rules:
+      - operations: [ "CREATE", "UPDATE", "DELETE" ]
```

Comment:
We would not need `DELETE` here right? I am assuming for namespaced groups we also don't have strict child group and ANP reference validations